src/PrintCore/PDEPlugInCallbackProtocol.cs

```diff
@@ -0,0 +1,119 @@
+#nullable enable
+
+#if MONOMAC
+
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+using ppd_file_s = System.IntPtr;
+
+#if !NET
+using NativeHandle = System.IntPtr;
+#endif
+
+namespace PrintCore {
+
+	public partial class PDEPlugInCallbackProtocol {
+
+#if NET
+		[SupportedOSPlatform ("macos13.0")]
+		[UnsupportedOSPlatform ("ios")]
+		[UnsupportedOSPlatform ("maccatalyst")]
+		[UnsupportedOSPlatform ("tvos")]
+#endif
+		public PMPrintSession? PrintSession {
+			get => GetPtrToStruct<PMPrintSession> (_GetPrintSession ());
+		}
```

Comment:
I am trying to do something similar to this in MetalPerformanceShaders: https://github.com/xamarin/xamarin-macios/blob/main/src/metalperformanceshaders.cs#L2800-L2809

Does anyone know why I am not able to call my `_GetPrintSession()` and other methods from my binding file here?

Reply:
`PDEPlugInCallbackProtocol` is a Protocol not a class, this will get generated as an interface, you are creating your own class here with `public partial class PDEPlugInCallbackProtocol {` so that class does not contain the `_GetPrintSession` method